cysync/src/store/database/helper/transaction/insert.ts

```diff
@@ -528,7 +528,7 @@ export const insertFromBlockbookTxn = async (transaction: {
         outputs: []
       };
 
-      await transactionDb.insert(feeTxn);
+      return feeTxn;
```

Comment:
In this case both the fees and the txn below should be added, which will not happen after this change.

Reply:
Nice catch!